keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -2180,11 +2066,39 @@ message NodePrepareResourcesResponse {
     // will be called again for those that are missing.
     map<string, NodePrepareResourceResponse> claims = 1;
 }
+
+message NodePrepareResourceResponse {
+    // These are the additional devices that kubelet must
+    // make available via the container runtime. A claim
+    // may have zero or more requests and each request
+    // may have zero or more devices.
+    repeated Device cdi_devices = 1 [(gogoproto.customname) = "CDIDevices"];
+    // If non-empty, preparing the ResourceClaim failed.
+    // cdi_devices is ignored in that case.
+    string error = 2;
+}
+
+message Device {
+    // A single device instance may map to several CDI device IDs.
+    // None is also valid.
+    repeated string cdi_ids = 1;
+
+    // The request in the claim that this device is associated with.
+    string request_name = 2;
+
+    // A single request may be satisfied by allocating several
+    // different devices. This is the index number starting at zero.
+    int index = 3;
 ```
 
+The `request_name` here allows kubelet to find the right CDI IDs for a
+container which uses device(s) from a specific request instead of all devices
+of a claim. The index gets added in preparation for supporting more
+than one device per request.
+
```

Comment:
Do we need to update the last sentence in light of including the `Amount` field in a request now?

Reply:
Yes, or drop the index in favor of implicit indexing (https://github.com/kubernetes/enhancements/pull/4709/files#r1633304279).

Reply:
Removed because @johnbelamaric suggested that we should not support index as selector for containers.